README.md

```diff
@@ -3,21 +3,91 @@
 # Welcome to Sign
 
 ## Brief intro to the repo 
-This is a Client Server Application to upload images to server/ S3 bucket
+This is an admin dashboard application for a school attendance marking system. Essentially, a Client Server Application to upload images to server/ S3 bucket. 
 
 ## Getting Started
-Learn how to setup this project and run it on your local machine.
+This is a simple PHP application. Learn how to setup this project and run it on your local machine. 
+
+### How to clone the repo 
+- On GitHub, navigate to the main page of the repository.
+- Under the repository name, click Clone or Download and copy the URL.
+- Open Terminal.
+- Change the current working directory to the location where you want the cloned directory to be made.
+- Type git clone, and then paste the URL you copied in Step 2.
+  `$ git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY`
+- Press Enter. Your local clone will be created at: 
+  `$ git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY`
+
+Read more here: (cloning-a-repository)[https://help.github.com/en/articles/cloning-a-repository]
+
+### How to run on the local apache server
+ - For Mac: (osxdaily.com/start-apache-web-server/)[http://osxdaily.com/2012/09/02/start-apache-web-server-mac-os-x/]
+ - For Windows: (msdn.microsoft.com/library/)[https://msdn.microsoft.com/en-us/library/ms181052(v=vs.80).aspx]
+ - For something else: (gist.github.com/jgravois/)[https://gist.github.com/jgravois/5e73b56fa7756fd00b89]
+
+If you've never contributed before, see [the first timer's guide on our blog](https://auth0.com/blog/a-first-timers-guide-to-an-open-source-project/) for resources and tips on how to get started.
```

Comment:
This is good. But better if this can be made short and sweet. Give the exact commands to clone this repo.

And setting up in local server. Better to give commands rather than links.

Reply:
the local server would differ for everyone 🤔